app/javascript/dashboard/store/modules/conversations/actions.js

```diff
@@ -405,6 +405,21 @@ const actions = {
     }
   },
 
+  updateDraftMessage: async (
```

Comment:
We have a store called draftMessages. Let us use that. 

https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/store/modules/draftMessages.js